server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -773,6 +767,27 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
 
     }
 
+    @Nullable
+    private AlertVO createAlertForTrivialAlertType(AlertType alertType, long dataCenterId, Long podId, Long clusterId) {
```

Comment:
`create` sounds a bit off since we get the alert entry from DB

Reply:
```suggestion
    private AlertVO getAlertForTrivialAlertType(AlertType alertType, long dataCenterId, Long podId, Long clusterId) {
```